config/src/test/java/org/springframework/security/config/annotation/web/configurers/HttpSecurityRequestMatchersTests.java

```diff
@@ -175,6 +211,70 @@ public void loadConfig(Class<?>... configs) {
 		this.context.getAutowireCapableBeanFactory().autowireBean(this);
 	}
 
+	@EnableWebSecurity
+	@Configuration
+	@EnableWebMvc
+	static class MultiMvcMatcherInLambdaConfig {
+
+		@Bean
+		@Order(Ordered.HIGHEST_PRECEDENCE)
+		SecurityFilterChain first(HttpSecurity http) throws Exception {
+			http.requestMatchers(
```

Comment:
Please use `//@ formatter:off` and `:on` so that the `http` formatting is preserved like so:

```java
http
    .requestMatchers((requests) -> requests
        .mvcMatchers("/test-1")
        .mvcMatchers("/test-2")
        .mvcMatchers("/test-3")
    )
    .authorizeRequests((authorize) -> authorize
        .anyRequest().denyAll()
    )
    .httpBasic(Customizer.withDefaults());
```